chef/cookbooks/network/recipes/default.rb

```diff
@@ -509,4 +511,18 @@ def kill_nic(nic)
       end
     end
   end
+
+  # If we're using an ovs-bridge somewhere, enable wicked-nanny to be started
+  # for the next boot.
+  # Note: There's no need to restart wicked here as all interfaces
+  # should be correctly configure at this point.
+  template "/etc/wicked/local.conf" do
+    source "wicked-local.conf.erb"
+    owner "root"
+    group "root"
+    mode "0644"
+    variables(
+      use_nanny: ovs_setup_once
```

Comment:
Just checking: there's no plan to restart wicked when that file changes, or is there? That was breaking HA when we had that.